tools/update_buildinfo.sh

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bash
```

Comment:
[ShellCheck](https://www.shellcheck.net/) reports a couple of issues in this script maybe worth fixing.

Reply:
fixed